Edit column names in bigquery table.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
+This is a step-by-step guide to help you change the name of a column in bigquery tables.
+
+## Step 1: Creating table with new schema in bigquery
+---
+In the **explorer** in bigquery in the dataset. Click on the ```⋮```to the right of the dataset name. And select the item **Create table**.  
+![create table](https://user-images.githubusercontent.com/38936255/146343440-b3db5c50-19b9-4314-9c7e-f82ca012d870.png)  
+And create a new table using the schema with the changed name of the field or fields.  
+At the end of the first step, you should have two tables. One original, the second with a modified scheme.  
+![img](https://user-images.githubusercontent.com/38936255/146344509-2aecf8bb-119c-455b-ba44-a95924bcc532.png)  
+## Step 2:  Import data from a table using the old schema into a new
+---
+Importing data into a new table will be done using a query():  
+```
+SELECT
+ * EXCEPT(column_one, column_two),
+ column_one AS newcolumn_one, column_two AS newcolumn_two
+FROM
+ mydataset.my_new_schema_table
+```
+After creating query with your initial data. It is necessary to specify in the query settings to which table the data will be transferred. To do this, need to go to the **Query Settings**.  
+
+![img](https://user-images.githubusercontent.com/38936255/146346033-33474622-b910-4315-8eb0-cd7f2e95dd50.png)
+
+In the settings will need in the **Destination** tab:  
```

Comment:
This step is not 100% clear for me, took me a few reads to get this, maybe something like:

`Perform following steps under **Destination** tab:`

Reply:
Agree with you. It looks more understandable. Edited: https://github.com/NebraLtd/hm-diag/blob/94bf73eec3ada1aeee700751fce0a00ba53ea390/Edit%20column%20names%20in%20bigquery%20table.md?plain=1#L25